all-in-one/BUILDING-TESTING.md

```diff
@@ -0,0 +1,68 @@
+# All-in-One Deployment Building & Testing
+
+## Building the Image Locally
+
+The Docker image can be built locally, instead of being pulled from the registry, by using the
+`Dockerfile` provided within this repository. Optionally, you can provide prebuilt copies of the
+[OSCAL Viewer][react-lib] and the [OSCAL REST Service][service]; however, if those are not provided
+the `Dockerfile` will automatically fetch the source from GitHub and build them when building the
+container.
+
+The latest tagged version of the [OSCAL Viewer][viewer] is available as a
+[build artifact][viewer-artifact] of the latest release of the [`oscal-react-library`][react-lib].
+You may download that file, pass it as the `VIEWER_PATH` build arg as described below. The latest
+snapshot release of the [OSCAL REST Service][service] is hosted on [GitHub packages][package].
```

Comment:
I had just copied and pasted from `README.md`. `BUILDING-TESTING.md` includes the information we didn't want to include in the Docker Hub description.

Reply:
A lot of this text was added in https://github.com/EasyDynamics/oscal-editor-deployment/pull/74/files#diff-6facdde444a2bf6f5b8d51238268835e50add0c2f96d053511a99ef4b51e3529. @mikeisen1 if you have specific suggestions for revising can you add them as suggestions on this PR or open a separate PR after this one is merged to help clean that up a bit? I agree it could use some refinement.

Reply:
I can open a separate PR on Monday.